src/main/resources/org/jenkins_ci/plugins/run_condition/core/StringsMatchCondition/config.jelly

```diff
@@ -25,6 +25,9 @@
 
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
 
+    <f:entry title="${%shellVariables}" field="enviornmentVariables">
```

Comment:
Spelling errors last a very long time in identifiers.  Could you correct this one?
```suggestion
    <f:entry title="${%shellVariables}" field="environmentVariables">
```

Reply:
@MarkEWaite Done sir